### DISCUSSION TITLE
Crazyflie 2.1 M2 blue light and unable to reflash firmware

### DISCUSSION THREAD

**Discussion by YWprog** [bitcraze/discussions#1384] [Q&A] [answered]
My crazyflie was able to work and connection few days ago, it suddenly fail to work. It could turn on with successful self-test at the beginning, after few times firmware failed to flash into it, it failed self-test. The crazyflie kept M2 blue light on, and others didn't, yet it could still be in the bootloader mode with M2 and M3 blue light blinking. On the other hand, both CrazyRadio and DFU are failed to reflash. I've seen another discussion implied that M2 blue light indicated the error of STM32, yet I tried several times but problem still. Is there any suggestion to solve this kind of problem? Thanks in advance!!

**Comment by gemenerik** [comment, marked as answer]
Try manually putting the Crazyflie in bootloader mode by holding the power button while it's off. Then in cfclient, in connect->bootloader->cold boot(recovery) tab->initiate bootloader->program to reflash.

If it still fails can you post some terminal printouts of the failed flash attempts?

**Reply by YWprog** [reply]
Thanks for your reply, the problem was perfectly solved with cold boot solution. Thank you again!!
